srcpkgs/Lidarr/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'Lidarr'
+pkgname=Lidarr
+version=0.5.0.583
+revision=1
+make_dirs="
+ /var/log/Lidarr/ 700 lidarr lidarr
+ /var/lib/lidarr/ 750 lidarr lidarr"
+hostmakedepends="msbuild-bin mono nodejs-lts git yarn"
+makedepends="libgdiplus-devel"
+depends="mono libmediainfo"
+short_desc="Music collection manager for Usenet and BitTorrent users"
+maintainer="Young Jin Park <youngjinpark20@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="http://lidarr.audio/"
+distfiles="https://github.com/lidarr/Lidarr/archive/v${version}.tar.gz"
+checksum="58b814975b568fccaf3f68bfc37b9c7d81bea123a5c5f4e57fa1fd08b16bf450"
+system_accounts="lidarr"
+lidarr_homedir="/var/lib/lidarr"
+
+post_extract() {
+	cd src/ExternalModules
+	git clone https://github.com/Sonarr/CurlSharp
```

Comment:
add it to distfiles

Reply:
They need to latest version within their repo that's not a relesae version. I guess I should request it?